SpatialGDK/Source/Public/Schema/UnrealMetadata.h

```diff
@@ -69,19 +70,23 @@ struct UnrealMetadata : Component
 	SubobjectToOffsetMap SubobjectNameToOffset;
 };
 
-FORCEINLINE SubobjectToOffsetMap CreateOffsetMapFromActor(AActor* Actor)
+FORCEINLINE SubobjectToOffsetMap CreateOffsetMapFromActor(AActor* Actor, FClassInfo* Info)
```

Comment:
If every Unreal instance can deterministically know static subobject offsets, is there any reason for us to put the `SubobjectToOffsetMap` on the `UnrealMetadata` anymore?